addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -20,6 +20,96 @@
 class AccountEdiFormat(models.Model):
     _inherit = 'account.edi.format'
 
+    # -------------------------------------------------------------------------
+    # Helpers
+    # -------------------------------------------------------------------------
+
+    @api.model
+    def _l10n_it_edi_generate_electronic_invoice_filename(self, invoice):
+        '''Returns a name conform to the Fattura pa Specifications:
+           See ES documentation 2.2
+        '''
+        a = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        n = invoice.id
+        progressive_number = ""
+        while n:
+            (n,m) = divmod(n,len(a))
+            progressive_number = a[m] + progressive_number
+
+        return '%(country_code)s%(codice)s_%(progressive_number)s.xml' % {
+            'country_code': invoice.company_id.country_id.code,
+            'codice': invoice.company_id.l10n_it_codice_fiscale.replace(' ', ''),
+            'progressive_number': progressive_number.zfill(5),
+        }
+
+    def _l10n_it_edi_check_invoice_configuration(self, invoice):
+        errors = []
+        seller = invoice.company_id
+        buyer = invoice.commercial_partner_id
+
+        # <1.1.1.1>
+        if not seller.country_id:
+            errors.append(_("%s must have a country") % (seller.display_name))
```

Comment:
```suggestion
            errors.append(_("%s must have a country", seller.display_name))
```
(and same comment for the other translations below)